nuvlaedge/agent/nuvla/client_wrapper.py

```diff
@@ -297,9 +297,9 @@ def telemetry(self, new_status: dict, attributes_to_delete: list[str]) -> dict:
         logger.debug(f"Sending telemetry report to Nuvla: \n"
                      f"Changed fields: {new_status}\n"
                      f"Deleted fields: {attributes_to_delete}")
-        response: CimiResource = self.nuvlaedge_client.edit(self.nuvlaedge_status_uuid,
-                                                            data=new_status,
-                                                            select=attributes_to_delete)
+        response: CimiResource = CimiResource(self.nuvlaedge_client._cimi_put(self.nuvlaedge_status_uuid,
+                                                                 data=new_status,
+                                                                 params={'select': attributes_to_delete}))
```

Comment:
This prevent the `get` operation done [here](https://github.com/nuvla/python-library/blob/master/nuvla/api/api.py#L491) but I think this operation is useless so this should probably be done in `python-library` instead of here.
@ignacio-penas what do you think ?

Reply:
I agree 100%. This was faster to implement in place and test.
I don't know if it would affect any other dependency that uses the edit function

Reply:
I don't think it will affect any other usage of the `edit` function because the `get` function into it is, AFAIK, useless.

Reply:
Understood. I will create the fix in job-engine.

Reply:
Here https://github.com/nuvla/python-library/pull/43